test/unit/resources/aws_billing_backend.rb

```diff
@@ -0,0 +1,99 @@
+module MockAwsBillingReports
+  class Empty < AwsBackendBase
+    def describe_report_definitions(_query)
+      Aws::CostandUsageReportService::Types::DescribeReportDefinitionsResponse.new(report_definitions: [])
+    end
+  end
+
+  class Basic < AwsBackendBase
+    def describe_report_definitions(_query)
+      Aws::CostandUsageReportService::Types::DescribeReportDefinitionsResponse
+        .new(report_definitions:
+      [
+        Aws::CostandUsageReportService::Types::ReportDefinition.new(
+          report_name: 'inspec1',
+          time_unit: 'HOURLY',
+          format: 'textORcsv',
+          compression: 'ZIP',
```

Comment:
Should this and below be lowercase `zip`/`gzip`?

Reply:
All caps is correct per https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/describe-report-definitions.html